packages/slice-machine/lib/builders/SliceBuilder/Header/index.tsx

```diff
@@ -55,91 +70,62 @@ const Header = ({ Model, store, variation, onSave, onPush, isLoading }) => {
                 </Link>
                 <Box sx={{ fontWeight: "thin" }} as="span">
                   <Text ml={2}>
-                    /{" "}
-                    {
-                      // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
-                      Model.infos.sliceName
-                    }{" "}
-                    /{" "}
-                    {
-                      // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
-                      variation.name
-                    }
+                    {`/ ${Model.infos.sliceName} / ${variation.name}`}
```

Comment:
why do you need string interpolation ? wasn't it good before ?

Reply:
same thing, just easier to read

Reply:
ok 👍 , did you know that it's prettier that adds the `{" "}` in the tsx XD ?